build/build-docker-deploy.sh

```diff
@@ -7,6 +7,12 @@ set -euo pipefail
 
 source $(dirname $0)/build-common.sh
 
+function path_for_docker() {
```

Comment:
I'm not that fond of this function name. Rather than a function, you could just declare a global variable:

```
# Relative path from the top-level cockroach directory to the current script.
relative_path_to_self="$(basename $(cd $(dirname $0); pwd))/$(basename $0)"
```

This also removes any concern about whether `$0` refers to script args or function args. (I would have thought script args, so I'm not actually sure how this is working).